integration/travis/run_integration.sh

```diff
@@ -1,10 +1,24 @@
 #!/bin/bash
+
+# Relies on the COOK_EXECUTOR environment variable to choose which edn file to load.
+# When set to 1, the Cook executor is enabled.
+# Else, the Mesos Command executor is used.
+
 set -ev
 
 NOSE_ATTRIBUTES=${1:-'!explicit'}
 
 export PROJECT_DIR=`pwd`
 
+CONFIG_FILE="scheduler_config_command_executor.edn"
+if [ "${COOK_EXECUTOR}" = "1" ]
+then
+  CONFIG_FILE="scheduler_config_cook_executor.edn"
+fi
```

Comment:
The issue is that we don't want an entry for `:executor` at all when the variable is set to false.

Also, the `scheduler_config_command_executor.edn` file is temporary until we switch to the cook executor completely.